lib/src/test/java/uk/gov/di/ipv/cri/address/library/service/AddressServiceTest.java

```diff
@@ -143,4 +151,330 @@ void shouldGetAddressItem() {
         addressService.getAddressItem(SESSION_ID);
         verify(mockDataStore).getItem(String.valueOf(SESSION_ID));
     }
+
+    @Test
+    void testAddressesLinkerNoAddressesPasses() {
+        List<CanonicalAddress> canonicalAddresses = new ArrayList<>();
+        assertDoesNotThrow(() -> addressService.setAddressValidity(canonicalAddresses));
+    }
+
+    @Test
+    void testLinkerIsCurrentAddressWithNULLNULLEvaluatesToPrevious() {
```

Comment:
remove the word `Linker` & `test` prefix from the unit test method names.
Rename the methods to start with `should` like the other unit tests in this file